### PR TITLE
Added plugin keyrecord

### PR DESCRIPTION
**Pull request by tunyash**
It allows user to record some keystrokes combination and replay it, for example change something minor in five places. That simple feature was in the wishlist for plugins, I've implemented it in the most simple way.

### CODE REVIEW

File: keyrecord/src/Makefile.am [hunk 1]
```diff
@@ -0,0 +1,10 @@
+include $(top_srcdir)/build/vars.build.mk
+plugin = autoclose
+
+geanyplugins_LTLIBRARIES = autoclose.la
+
+autoclose_la_SOURCES = autoclose.c
+autoclose_la_CPPFLAGS = $(AM_CPPFLAGS) -DG_LOG_DOMAIN=\"AutoClose\"
+autoclose_la_LIBADD = $(COMMONLIBS)
```
Comment (frlan) [inline keyrecord/src/Makefile.am]: Please check for right prefix -> s/autoclose/keyrecord

File: keyrecord/src/keyrecord.c [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+	}
+
+}
+
+void geany_load_module(GeanyPlugin *plugin)
+{
+	/* main_locale_init() must be called for your package before any localization can be done */
+	main_locale_init(LOCALEDIR, GETTEXT_PACKAGE);
+	plugin->info->name = _("Keystrokes recorder");
+	plugin->info->description = _("Allows to record some sequence of keystrokes and replay it");
```
Comment (frlan) [inline keyrecord/src/keyrecord.c]: This needs adding files to po/POTFILES.in

**Comment by frlan** [comment]
Please remove README~ from pull request as well as ensure MAINTAINER-entry for your plugin is set.

File: keyrecord/src/keyrecord.c (hunk 2 above)
Comment (tunyash) [inline keyrecord/src/keyrecord.c]: Done, thank you.

**Comment by b4n** [comment]
Out of curiosity, what is the functional difference with GeanyMacro?

**Comment by tunyash** [comment]
Geany macro is more about commonly used macros, this plugin is about help repeating actions for several times.

**Comment by frlan** [comment]
Sorry for leaving this so long here. I just don't get it working.Can you check wehther its not yet broken and maybe improve README?

**Comment by frlan** [comment]
Ahh. Beside of this seem to not break anything. Merging it

**Pull request merged**